src/cfnlint/data/AdditionalSpecs/OnlyOne.json

```diff
@@ -38,5 +38,25 @@
     ]
   },
   "ResourceTypes": {
+    "AWS::AutoScaling::AutoScalingGroup": [
+      [
+        "InstanceId",
+        "LaunchConfigurationName",
+        "LaunchTemplate",
+        "MixedInstancesPolicy"
+      ]
+    ],
+    "AWS::Route53::RecordSet": [
```

Comment:
Same.

`Required: Conditional. You must specify either the HostedZoneName or HostedZoneId, but you cannot specify both. If this record set is part of a record set group, do not specify this property.`

Reply:
This is correct, right? "but you cannot specify both", that makes it "only one"
🤔

Reply:
Yea, I'm wondering if it should be in both files  AtLeastOne and OnlyOne

Reply:
But this is then correct right? "You must specify either the HostedZoneName or HostedZoneId, but you cannot specify both"

Reply:
yes this is also correct (from what I can tell).